apps/admin-gui/src/app/shared/components/dialogs/edit-application-form-item-dialog/edit-application-form-item-dialog.component.ts

```diff
@@ -249,6 +257,42 @@ export class EditApplicationFormItemDialogComponent implements OnInit {
     to.hiddenDependencyItemId = from.hiddenDependencyItemId;
   }
 
+  private prepareFormControls(): void {
+    const formGroupFields: { [key: string]: FormControl } = {};
+    for (const lang of this.languages) {
+      // Plain
+      formGroupFields[`${lang}-plain-error-message`] = new FormControl(
+        this.applicationFormItem.i18n[lang].errorMessage,
+        []
+      );
+      formGroupFields[`${lang}-plain-help`] = new FormControl(
+        this.applicationFormItem.i18n[lang].help,
+        []
+      );
+
```

Comment:
Shouldn't these be only added for non HTML fields? It seems the HTML forms need just `lang-html-label` FormControl, while the plain text forms need `lang-plain-label`, `lang-plain-error` and `lang-plain-help`. Based on the content that is shown on withing the tab. Unless I am missing something.